Scripts/Run-SS-On-Exp&Sim/scripts/fourier_descriptor.py

```diff
@@ -0,0 +1,95 @@
+import matplotlib.pyplot as plt
+import image_processing
+import BacteriaFeatures
+import numpy as np
+
+
```

Comment:
It would be helpful to include a brief summary of what this script does. Something like the comment in the commit message would be appropriate.

Make sure you include a doc string in each function.